interp/notation.ml

```diff
@@ -1349,6 +1349,34 @@ let entry_has_ident = function
   | InCustomEntryLevel (s,n) ->
      try String.Map.find s !entry_has_ident_map <= n with Not_found -> false
 
+let entry_has_numeral_map = ref String.Map.empty
```

Comment:
This state is not undone in backtracking? This seems wrong.

Reply:
I guess the right way to avoid this problem is to make all the data into a record and have `freeze / unfreeze` below just restore the record; actually I have a tree that started to do that, but I don't remember what the problem was.

Reply:
Right, registration was missing. I now made a record of all properties of a custom entry and fixed that. Thanks.